interview-questions.md

```diff
@@ -9,49 +9,127 @@ Then, research the question to expand on your answer. Even if you feel you have
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 Your answer:
+Conditional statements are commands for handiling decisions within a programming language. All conditional statements must have an "If" statement. The logic behind conditional statments say - if "this" is true, do this action, If not do this other action instead.
 
 Researched answer:
+Conditional statements control behavior in a programming language and determine whether or not a piece of code can be executed.There ares several different types of conditionals in JavaScript. IF statements dertermine if a condition is true. It is used to specify execution of a particular action for in a piece of code ELSE IF statements determine if the same condition is false. If so, the code will specifiy different action to perform instead. ELSE statement specifify a  if the first condition is false.
+(Create JavaScript Conditionals with This JavaScript Resource, n.d.)
+
+
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a document control system designed to track revisions of code.
```

Comment:
Make sure to treat these questions as if you were asked this in an interview.  One-line answers won't give you the opportunity to really share much about your knowledge with your interviewer.  Extend these answers by talking about all parts of the question, use definitions, and provide examples if possible.